tests/test_data/test_datasets/test_generation_datasets.py

```diff
@@ -0,0 +1,261 @@
+from pathlib import Path
+
+import pytest
+
+# yapf: disable
```

Comment:
Can yapf be enabled?

Reply:
It will conflict with `isort`

Reply:
Yes isort is a stubborn package, and it often conflicts with yapf.

As a fix, please copy the yapf config section here https://github.com/open-mmlab/mmpose/blob/202983d24665a909ae1c45f4025d66794b9e32fd/setup.cfg#L10 and try again.

Really hope there's a better alternative for isort. Before that, pls try the above method and enable yapf